public/docs/_examples/server-communication/dart/lib/wiki/wikipedia_service.dart

```diff
@@ -0,0 +1,24 @@
+// #docregion
+import 'dart:async';
+import 'package:angular2/angular2.dart';
+import 'package:jsonpadding/jsonpadding.dart';
```

Comment:
@zoechi `package:jsonpadding` ? That's a library I created to handle the jsonp calls since the other [jsonp](https://pub.dartlang.org/packages/jsonp) library was outdated and overcomplicated, the maintainer said it was overcomplicated because when he first built it `dart:js` wasn't very mature. I asked him to update it and seems like he did, I'll check if it is usable now.
I created this one but it has no testing yet, I've been busy these days.